flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/batch/table/CorrelateITCase.scala

```diff
@@ -82,6 +82,42 @@ class CorrelateITCase(
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
 
+  /**
+    * Due to CALCITE-2004, common join predicates are temporarily forbidden.
+    */
+  @Test (expected = classOf[ValidationException])
+  def testLeftOuterJoinWithPredicates(): Unit = {
```

Comment:
The validation has been checked before. We don't need an ITCase for this.

Reply:
This test should have been used as a normal test case for this join, i.e., we don't except it to raise an exception. I'd suggest to keep it for later use, otherwise we may not be aware the test is incomplete in the future.